.github/workflows/wheel.yml

```diff
@@ -0,0 +1,28 @@
+# Builds a binary distibutable package.
+# Minimal functionality checks may be run as part of the build process, but
+# source code checks and extensive functionality checks (e.g. tox) belong in presubmit.yml
```

Comment:
Is it worth testing installation of the wheel in a new environment?  I think that test would be useful.  Thoughts?  This could be added onto #28?

Reply:
Maybe. I'll see if I can add something simple to https://github.com/SymbiFlow/fasm/pull/28.